### DISCUSSION TITLE
Where do you deploy Kubernetes? Pricing and performance

### DISCUSSION THREAD

**Discussion by andershagbard** [actions/actions-runner-controller#2278] [General]
In my team, we are currently deploying Our ARC Kubernetes on Digital Ocean, through their dedicated Kubernetes setup. I feel like the performance is slow, compared to the price ($36)

I have 2x Basic - 2vCPU 2 GB RAM.

Is this enough? Is there a better alternative?

**Comment by kevholmes** [comment]
Is that $36/month? What about Linode? Looks like you can get a shared 4 CPU / 8 GB RAM hosted k8s setup for $40/month: https://cloud-estimator.linode.com/s/

**Reply by kevholmes** [reply]
To clarify, I do not use Linode for this but have used their services for many years in other companies without many complaints. I am currently using ARC in a Private GKE cluster and it's quite good.
